frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -80,7 +80,7 @@ export const XYChart = ({
     xTicksDimensions.width,
     settings.labels,
     style.axes.ticks.fontSize,
-    !!settings.goal,
+    !!settings.goal || !!settings.show_values,
```

Comment:
normally I'd prefer `Boolean(expression)` but I use `!!` just to be consistent with the existing expression.